restler/unit_tests/log_baseline_test_files/Bug_Buckets_Json/InvalidValueChecker_500_1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "status_code": "500",
+    "checkerName": "InvalidValueChecker",
+    "reproducible": true,
+    "verb": "PUT",
+    "endpoint": "/item/{itemName}",
+    "status_text": "Internal Server Error",
+    "request_sequence": [
+        {
+            "requestOrder": 0,
+            "replayrequest": "PUT /item/itemName<test!>-bc8aff2b56?location=centralus&date=2021-01-01 HTTP/1.1\r\nAccept: application/json\r\nHost: unittest\r\nContent-Type: application/json\r\nAUTHORIZATION TOKEN\r\n\r\n{\"datetest\":\"2021-01-01\",\"id\":\"/testparts/{BT\t[\"}\r\n",
```

Comment:
>replayrequest [](http://example.com/codeflow?start=13&length=13)

how about `replay_request` for readability?

Reply:
updated